src/mame/casio/pv1000.cpp

```diff
@@ -266,6 +270,24 @@ void pv1000_state::io_w(offs_t offset, uint8_t data)
 		m_pcg_bank = (data & 0xe0) >> 5;
 		m_force_pattern = ((data & 0x10) >> 4); /* Dig Dug relies on this */
 		m_render_disable = ((data & 0x08) >> 3);
+		if (m_render_disable == 0) // If we're enabling rendering mid-scanline...
+		{
+			int hpos = m_screen->hpos();
+			int vpos = m_screen->vpos();
+			hpos += 32; // adjust from screen coordinates to something that's easier to math
```

Comment:
MAME historically starting at origin 0,0 is just for a generic stub convenience.
This kind of math is unnecessary if you acknowledge overscan origins in `screen_update` and `set_raw` setups.

Reply:
To further clarify, if you do set_raw with the real hcount/vcount values you also have to change where you draw in screen_update to coincide with the specified visible area.  I get *why* MAME does that, but it really does violate the principle of least surprise.